app/styles/components/image-preview.scss

```diff
@@ -0,0 +1,22 @@
+@import '../variables';
+
+.image-preview {
+  position: relative;
+
+  .preview {
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 200px;
+    margin: $padding-m;
+  }
+
+  .remove {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.